### DISCUSSION TITLE
How to make cron jobs on inserting new row in table

### DISCUSSION THREAD

**Discussion by kresogalic8** [supabase/supabase#17141] [Questions]
Hey guys, can someone explain how to use pg_cron and how to run cron job after inserting the data in some table? Just for example, i create a maintenance record in table and i want to repeat (insert in the invoices table) the invoice each month for that maintenance, how can i do that?

**Comment by GaryAustin1** [comment]
You can set up a cron task from a trigger function as long as it is security definer so it has the privilege to do so.
You might be better off though having a single cron task that selects from your other table based on the row having a "renewal" time that has passed, then reset the time for next month.
